### ISSUE TITLE
training

### ISSUE THREAD

**Issue by fousdfrf** [KangJialiang/ViSpec#10]
Regarding training, I'd like to ask for your advice: In my training setup, I'm using a two-stage training strategy with a batch size (bs) of 8 and a learning rate (lr) of 3e-6. The resulting accuracy is only 68%—is this normal?

<img width="792" height="194" alt="Image" src="https://github.com/user-attachments/assets/aea37447-5bb5-4f7c-8088-13ef35c5f41c" />

Additionally, when I increase the batch size to 16, how should I adjust the learning rate? I tried increasing the learning rate to 5e-5, which worked fine when training on text-only data. However, when training on multimodal data, the accuracy peaks at around the 10th epoch and then starts to decline, eventually dropping to zero.
